dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json

```diff
@@ -319,26 +319,26 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "ray_resources{Name=\"object_store_memory\",State=\"USED\"} / 1e9",
+          "expr": "sum(ray_object_store_memory / 1e6) by (Location)",
           "interval": "",
           "legendFormat": "Object Store: {{instance}}",
           "queryType": "randomWalk",
           "refId": "A"
         },
         {
           "exemplar": true,
-          "expr": "sum(ray_resources{Name=\"object_store_memory\"} / 1e9)",
+          "expr": "",
```

Comment:
Hmm we still want the MAX query. Can you put this back?

Reply:
Yeah - I agree.

Reply:
Kind of messed up the json.